pkg/diag/colors/colors.go

```diff
@@ -117,10 +117,15 @@ var (
 	BrightMagenta = Command("fg 13")
 	BrightCyan    = Command("fg 14")
 
+	RedBackground    = Command("bg 1")
+	GreenBackground  = Command("bg 2")
+	YellowBackground = Command("bg 3")
+	BlueBackground   = Command("bg 4")
+
 	// We explicitly do not expose blacks/whites.  They're problematic given that we don't know what
 	// terminal settings the user has.  Best to avoid them and not run into contrast problems.
 
-	// Black         = Command("fg 0")
+	Black = Command("fg 0") //only use with background color
```

Comment:
Super nit: Please add a space after the `//` and comments that aren't just 1-3 words should be in complete sentences, beginning with a capital letter and ending with a period.